easybuild/easyconfigs/l/launcher/launcher-3.7-foss-2020a.eb

```diff
@@ -0,0 +1,51 @@
+# Autor:  Felipe Franco <felipe.franco@hpcnow.com>
+#
+easyblock = 'Tarball'
+
+name = 'launcher'
+version = '3.7'
+
+homepage = 'https://github.com/TACC/launcher'
+description = """
+Launcher is a utility for performing simple, data parallel, 
+high throughput computing (HTC) workflows on clusters, 
+massively parallel processor (MPP) systems, workgroups of computers, 
+and personal machines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
Hi @jfelipe-francogonzalez - normally only packages that are performance-sensitive and/or use MPI would sit at the highest level of the toolchain hierarchy, i.e. `foss` or `intel`. In this case, you can safely move it to `GCCcore`, version `9.3.0`.

Reply:
Done. Please see the merged PR below